config/routes.rb

```diff
@@ -9,7 +9,9 @@
 
```

Comment:
- [ ] It would be nice if you fix the order of the migrations because when I try to run the migrations then I get errors :fearful: because of the order :thinking: The order should be like

1. Create User Migration
2. Create Post Migration
3. Create Comment Migration
4. Create Like Migration
5. Other Migration

Reply:
- [ ] Just as the previous reviewer mentioned kindly fix your migration files because it is making your migration not run
![image](https://github.com/devalibello/blog-app/assets/66097371/017c687d-8665-409b-9c35-95c0d969c8e8)